backend/src/modules/authorization/router.js

```diff
@@ -64,12 +63,31 @@ router.post('/signUp', checkSchema({
   const errors = validationResult(req)
 
   if (!errors.isEmpty()) {
-    return res.json(errors.array())
+    return res.json({ errors: errors.array() })
   }
 
   const result = await Controller.signUp(req.body)
 
   return res.json(result)
 })
 
+router.post('/logOut', checkSchema({
+  authorization: {
+    in: 'body',
```

Comment:
authorization must not be in "body", but in "header";
there is isJWT option available for express-validator;